src/test/fuzz/script.cpp

```diff
@@ -4,10 +4,10 @@
 
 #include <chainparams.h>
 #include <compressor.h>
-#include <core_io.h>
 #include <core_memusage.h>
 #include <policy/policy.h>
 #include <pubkey.h>
+#include <rpc/blockchain.h>
```

Comment:
This is pretty ugly. The fuzz test shouldn't be using command-line options at all; just invoke the explicit-addresses version once with false and once with true, if you need to.